res/mac/FreenetTray/src/LICENSE

```diff
@@ -1,81 +1,45 @@
 Original code:
 
 This program is free software; you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation; either version 2 of the License, or
-    (at your option) any later version.
+it under the terms of the GNU General Public License version 2, 
+as published by the Free Software Foundation.
```

Comment:
Okay, here's a bit more:
http://www.gnu.org/licenses/gpl-faq.html#GiveUpKeys
https://www.gnu.org/licenses/gpl-faq.html#Tivoization

In summary, you're only forced to give up your keys if you are distributing hardware which checks signatures. So e.g. Apple can't use GPL3 code because they intend to do things the GPL3 forbids.

If this DID apply to us then it'd be very worrying, because presumably Apple would want the JVM to enforce it as well. As I've explained we can't make fred GPL-2-only without rewriting a lot of code.